docs/spec_crashreport.rst

```diff
@@ -69,6 +70,45 @@ Example::
    1
 
 
+Annotations as single JSON-encoded value
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+1. The ``Content-Disposition`` must be ``form-data``.
+
+2. The ``Content-Disposition`` must specify a ``name``. The value should
+   be ``extra``. The value must be in ASCII.
+
+3. The ``Content-Type`` must be ``application/json``.
+
+4. The value of this field must a JSON-encoded string of crash report
+   annotations.
+
+   Each annotation value must be a string. If the value is an object, it must
+   be JSON-encoded.
```

Comment:
It's hard to express this one in a way that's easy to understand. Maybe something like:
```
Annotations cannot contain JSON-encoded objects directly, they must be escaped so that they're stored as a single string.
```
I'm not sure if it's better though :-/